library/globals.inc.php

```diff
@@ -2208,128 +1967,20 @@
-      xl('URLs for NewCrop eRx Service Address, separated by a semi-colon.')
-    ),
-
-    'erx_soap_ttl_allergies' => array(
-      xl('NewCrop eRx SOAP Request Time-To-Live for Allergies'),
-       'num',
-       '21600',
-      xl('Time-To-Live for NewCrop eRx Allergies SOAP Request in seconds.')
-    ),
-
-    'erx_soap_ttl_medications' => array(
-      xl('NewCrop eRx SOAP Request Time-To-Live for Medications'),
-       'num',
-       '21600',
-      xl('Time-To-Live for NewCrop eRx Medications SOAP Request in seconds.')
-    ),
-
-    'erx_account_partner_name' => array(
-      xl('NewCrop eRx Partner Name'),
-       'text',
-       '',
-      xl('Partner Name issued for NewCrop eRx service.')
-    ),
-
-    'erx_account_name' => array(
-      xl('NewCrop eRx Name'),
-       'text',
-       '',
-      xl('Account Name issued for NewCrop eRx service.')
-    ),
-
-    'erx_account_password' => array(
-      xl('NewCrop eRx Password'),
-       'pass',
-       '',
-      xl('Account Password issued for NewCrop eRx service.')
-    ),
-
-    'erx_account_id' => array(
-      xl('NewCrop eRx Account Id'),
-       'text',
-       '1',
-      xl('Account Id issued for NewCrop eRx service, used to separate multi-facility accounts.')
-    ),
-
-    'erx_upload_active' => array(
-      xl('Only upload active prescriptions'),
-       'bool',
-       '0',
-      xl('Only upload active prescriptions to NewCrop eRx.')
-    ),
-
-    'erx_import_status_message' => array(
-      xl('Enable NewCrop eRx import status message'),
-       'bool',
-       '0',
-      xl('Enable import status message after visiting NewCrop eRx.')
-    ),
-
-    'erx_medication_display' => array(
-      xl('Do not display NewCrop eRx Medications uploaded'),
-       'bool',
-       '0',
-      xl('Do not display Medications uploaded after visiting NewCrop eRx.')
-    ),
-
-    'erx_allergy_display' => array(
-      xl('Do not display NewCrop eRx Allergy uploaded'),
-       'bool',
-       '0',
-      xl('Do not display Allergies uploaded after visiting NewCrop eRx.')
-    ),
+  'Lab' => array(
 
-    'erx_default_patient_country' => array(
-        xl('NewCrop eRx Default Patient Country'),
-        array(
-            '' => '',
-            'US' => xl('USA'),
-            'CA' => xl('Canada'),
-            'MX' => xl('Mexico'),
-        ),
-         '',
-        xl('Default Patient Country sent to NewCrop eRx, only if patient country is not set.'),
-    ),
 
-    'erx_debug_setting' => array(
-        xl('NewCrop eRx Debug Setting'),
-        array(
-            0 => xl('None'),
-            1 => xl('Request Only'),
-            2 => xl('Response Only'),
-            3 => xl('Request & Response'),
-        ),
-         '0',
-        xl('Log all NewCrop eRx Requests and / or Responses.'),
     ),
+  
+    // Mail Tab
+  //
+  'Mail' => array(
```

Comment:
I think Mail and SMS can be one tab.   Does the SMS stuff even work?  what's required to make it work?   It used to be MaviQ, but that is no longer functioning (I know 'cause it was an MI2 thing)